assets/styles/styles.scss

```diff
@@ -64,6 +67,9 @@
 @import "components/wysiwyg";
 @import "components/search";
 @import "components/subcategories";
+@import "components/free-text";
+@import "components/quote_and_image";
```

Comment:
Naming convention for this should follow the others (dashes instead of spaces). You will also need to change the file name.